### DISCUSSION TITLE
Fixing samples after sectioning

### DISCUSSION THREAD

**Discussion by mickelj7** [IBEXImagingCommunity/ibex_imaging_knowledge_base#135] [General]
Hello for the research I'm working on I need unfixed tissue sectioned for RNA processing. I am trying to do serial sectioning with the RNA section and was going to fix my sections after placing on slides. My question is, would this work for IBEX? My plan was to section on regular slides and fix the sections then continue with IHC. I'm not sure if I would be able to use the two well chamber cover glass but I was hoping if I do the staining manually and pool the solutions on the slide it would be sufficient. Any feed back or recommendations would be greatly appreciated.

**Comment by radtkea** [comment]
Hi Jaide,

Thank you very much for your question! You should be able to fix your section after placing on the slides. Once the sample are fixed, you can do IBEX. I would think it could work on slides as well as two well chamber cover glass. 

I will share your questions with some other community members who have more direct experience working with RNA and protein on the same tissue section. 

Stay tuned and thanks so much!
A

**Reply by mickelj7** [reply]
Thank you. I did have a specific question about fixing the section. I am hoping to do a lipid stain with Bodipy or if there's another option for lipid staining with the IBEX procedure. Is there a way to fix the section without dissolving the fat so I can stain it?

**Reply by radtkea** [reply]
Hi Jaide,

Thanks for your question!

I have never performed lipid staining on fixed samples. Summarizing my thoughts/hunches below:

1. I don't think Bodipy is recommended for fixed tissues. See their [FAQ](https://www.thermofisher.com/order/catalog/product/D3922?ef_id=Cj0KCQjwir2xBhC_ARIsAMTXk85T80ZS9BIzPIllnWE1XONtN8HiEQ0ase-Xqz6cC3qqTu88qbcF364aAo5oEALw_wcB:G:s&s_kwcid=AL!3652!3!447292198730!!!g!!!10506731179!109642167491&cid=bid_pca_iva_r01_co_cp1359_pjt0000_bid00000_0se_gaw_dy_pur_con&gad_source=1&gclid=Cj0KCQjwir2xBhC_ARIsAMTXk85T80ZS9BIzPIllnWE1XONtN8HiEQ0ase-Xqz6cC3qqTu88qbcF364aAo5oEALw_wcB) related to FFPE on the Bodipy product page.
2. Biotium offers many different fluorescent probes. I have not used their [LipidSpot](https://biotium.com/product/lipidspot-488-lipid-droplet-stain-1000x/) but they provide [detailed recommendations](https://biotium.com/faqs/can-the-lipidspot-lipid-droplet-stains-be-used-on-tissue-sections/) related to your question. 
3. This [thread](https://www.researchgate.net/post/Can_anyone_suggest_any_staining_for_lipids_by_using_formalin_fixed_tissue) might be helpful to you. I especially liked Peter Walker's comment for positive (mouse aorta with brown fat) and negative (tissue treated with acetone) controls.
4. Other options: [PhenoVue 493 Lipid Stain](https://www.revvity.com/product/phenovue-493-lipid-stain-cp51) and [3 methods](https://www.revealbio.com/post/precision-histology-steatosis-quantitative-analysis-of-abnormal-lipid-accumulation/).

In addition to the impact of fixation on lipids, I would make sure you do not have a detergent in your fixative or blocking/labeling buffers. Best of luck. Please keep us posted!

**Reply by radtkea** [reply]
Putting all your questions together, you might want to preserve your tissues as frozen but not fixed samples, preserving RNA and lipids. Then, you could try [Oil Red O Stain Kit](https://www.abcam.com/products/assay-kits/oil-red-o-stain-kit-lipid-stain-ab150678.html). The manufacturer says this kit may be used ONLY on frozen tissue sections, fresh smears, or touch preps. 

This paper says that [Oil Red O Stain](https://cris.maastrichtuniversity.nl/ws/files/75484061/Hesselink_2001_Optimisation_of_oil_red_O.pdf) can be examined by fluorescence microscopy using a 540- to 580-nm Texas red excitation/emission filter. They also provide a protocol with fixed tissue and comment on the incorporation of detergents in the staining protocol. 

You could do a post-fix, 10 minutes with 1% PFA as instructed by Zhixin, after lipid staining but before antibody labeling. Order of operations: Fresh frozen sample + lipid dye. Wash. Fix 10 minutes with 1% PFA. Wash. Antibody labeling.

**Comment by gokujianghu** [comment]
Hi Jaide,

I have worked with Andrea and tested fresh frozen lymph node and spleen sections for surface marker staining, all marker tested except for collagen did not work. However, when we did post fix with 1% PFA for 10min, and then stain for the surface markers, they all worked similarly to fixed frozen sections, but we have not tested RNA quality yet. 

Best,
Zhixin
